README.md

```diff
@@ -102,24 +103,26 @@ Below is a list of specs for the underlying base encodings:
 - `base58flickr` https://datatracker.ietf.org/doc/html/draft-msporny-base58-02, but using a different alphabet
 - `proquint` [Proquint RFC](rfcs/Proquint.md), which is the [original spec](https://arxiv.org/html/0901.4016) with an added prefix for legibility
 
-## Reserved
-
-The following codes are _reserved_ for (backwards) compatibility with existing systems.
 
-* `/` - Separator used by [multiaddr](https://github.com/multiformats/multiaddr).
-* `1` - Base58 encoded identity multihashes used by libp2p peer IDs.
-* `Q` - Base58 encoded sha2-256 multihashes used by libp2p/ipfs for peer IDs and CIDv0.
-
-If you'd like to switch a project over to multibase and would also like to
-reserve a prefix for compatibility, please file an issue.
 
 ## Status
 
 Each multibase encoding has a status:
 
-* draft - these encodings have been proposed but are not widely implemented and may be removed.
-* candidate - these encodings are mature and widely implemented but may not be implemented by all implementations.
-* default - these encodings should be implemented by all implementations and are widely used.
+* reserved - for functional reasons or to avoid collisions with other multi-* registries, this registry cannot accept registrations at this code-point and implementing one unregistered is discouraged for interoperability reasons
+* experimental - these encodings have been proposed but are not widely implemented and may be removed.
+* draft - these encodings are mature and widely implemented but may not be implemented by all implementations.
+* final - these encodings should be implemented by all implementations and are widely used.
+* deprecated - this entry will likely be removed and reassigned in the future and it will not likely become a `final` registration
+
+### Reserved Terms
+
+The following codes are _reserved_ and cannot be registered in the `multibase` table. Note that all three of the Unicode entries, expressed as the [unsigned varint] expression of that Unicode code-point in UTF-8, correspond to widely-used entries in the [multiformats registry group] that could create confusions for some legacy systems handling both binary and multibased structures from other multiformats. While technically the multibase registry is not part of the [multiformats registry group], these reservations minimize risk of confusion when composing multiple multiformats in one data system.
+
+* `NUL` (n/a) - Legacy data may be found with null-byte-prefixed binary structures mixed in among multibase-encoded ones in arrays of data, although support for this is no longer mandated by conformant implementations.
```

Comment:
I would use `U+0000` instead of `n/a`.

Reply:
LOL, after this being merged, I realized that `n/a` is actually more correct and they way to go. It was always the byte and not the code point.